combined.log

```diff
@@ -0,0 +1,33 @@
+
+{"level":"info","message":"---------------------------","service":"user-service"}
```

Comment:
can we ignore this file

Reply:
This file is used to log the results  above the "info" level using Winston logging library. (Including errors and content).